nva-datamodel-java/src/main/java/no/unit/nva/model/contexttypes/MediaContribution.java

```diff
@@ -53,6 +53,10 @@ public String getDisseminationChannel() {
         return disseminationChannel;
     }
 
+    public SeriesEpisode getPartOf() {
```

Comment:
Do you need to update equals and hash method?

Reply:
I see partOf is there, but should probably use getPartOf() to make it **beautiful**